### PR TITLE
Translation editor

### PR DESCRIPTION
**Pull request by eatyourgreens**
Adds a reusable `TranslationField` component for translating resource properties.

Adds a 'Translate' button for translating each field, and basic styling.

### CODE REVIEW

File: src/components/FieldGuide.jsx [hunk 1]
```diff
@@ -12,15 +12,24 @@ options.log = (test) => {
 };
 
 function FieldGuideItem(props) {
-  const { index, item } = props;
+  const { index, item, translation } = props;
```
Comment (simoneduca) [inline src/components/FieldGuide.jsx]: `index` should be added to `propTypes`

File: src/components/ProjectContents.jsx [hunk 2]
```diff
@@ -13,26 +13,40 @@ options.log = (test) => {
...
-        <h2>Translation</h2>
-        <p>Title: { translationContents.title }</p>
-        <p>Description: { translationContents.description }</p>
-        <Markdown content={`Introduction: ${translationContents.introduction}`} />
-        <p>Researcher quote: { translationContents.researcher_quote }</p>
-        <p>Language: { translationContents.language}</p>
-      </div>
+    <div>
+      <h2>Project</h2>
+      <TranslationField
```
Comment (simoneduca) [inline src/components/ProjectContents.jsx]: Linter's complaining about trailing spaces

**Comment by simoneduca** [comment]
I'm going to leave the branch in case you want to add `index` to `propTypes`

**Comment by eatyourgreens** [comment]
Fixed in 63d1b28, I think.

File: src/components/FieldGuide.jsx [hunk 3]
```diff
@@ -12,34 +12,58 @@ options.log = (test) => {
...
+        translation={translation.content}
+      >
+        Content
+      </TranslationField>
     </li>
   );
 }
 
 FieldGuideItem.propTypes = {
+  index: PropTypes.integer,
```
Comment (eatyourgreens) [inline src/components/FieldGuide.jsx]: `integer` isn't a valid prop type.

Comment (simoneduca) [inline src/components/FieldGuide.jsx]: Yes, it's supposed to be `number`. Sorry I missed that.

**Pull request merged**